.rubocop_todo.yml

```diff
@@ -0,0 +1,301 @@
+# SupportedStyles: final_newline, final_blank_line
+Layout/TrailingEmptyLines:
+  Exclude:
+    - 'lib/hammer_cli_foreman_discovery/discovery_references.rb'
+    - 'lib/hammer_cli_foreman_discovery/i18n.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: PreferredName.
+Naming/RescuedExceptionsVariableName:
+  Exclude:
+    - 'lib/hammer_cli_foreman_discovery/discovery.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Style/Encoding:
+  Exclude:
+    - 'hammer_cli_foreman_discovery.gemspec'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Style/ExpandPathArguments:
+  Exclude:
+    - 'hammer_cli_foreman_discovery.gemspec'
+
+# Offense count: 13
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: always, always_true, never
+Style/FrozenStringLiteralComment:
+  Exclude:
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'hammer_cli_foreman_discovery.gemspec'
+    - 'lib/hammer_cli_foreman_discovery.rb'
+    - 'lib/hammer_cli_foreman_discovery/command_extensions/provision_with_puppet.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery_references.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery_rule.rb'
+    - 'lib/hammer_cli_foreman_discovery/i18n.rb'
+    - 'lib/hammer_cli_foreman_discovery/version.rb'
+    - 'test/unit/discovery_resource_mock.rb'
+    - 'test/unit/discovery_rules_test.rb'
+    - 'test/unit/discovery_test.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Style/IfUnlessModifier:
+  Exclude:
+    - 'lib/hammer_cli_foreman_discovery/discovery.rb'
+
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: short, verbose
+Style/PreferredHashMethods:
+  Exclude:
+    - 'lib/hammer_cli_foreman_discovery/discovery.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowedCompactTypes.
+# SupportedStyles: compact, exploded
+Style/RaiseArgs:
+  EnforcedStyle: compact
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Style/RedundantSelf:
+  Exclude:
+    - 'Gemfile'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: implicit, explicit
+Style/RescueStandardError:
+  Exclude:
+    - 'lib/hammer_cli_foreman_discovery.rb'
+
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: RequireEnglish.
+# SupportedStyles: use_perl_names, use_english_names, use_builtin_english_names
+Style/SpecialGlobalVars:
+  EnforcedStyle: use_perl_names
+
+# Offense count: 2
+# This cop supports unsafe autocorrection (--autocorrect-all).
```

Comment:
`This cop supports unsafe autocorrection (--autocorrect-all).`

Can we fix the issues? It's a small plugin, would be nice to have it "clean"